src/rp2_common/pico_stdio_usb/stdio_usb.c

```diff
@@ -56,8 +56,17 @@ static int64_t timer_task(__unused alarm_id_t id, __unused void *user_data) {
 
 static void low_priority_worker_irq(void) {
     if (mutex_try_enter(&stdio_usb_mutex, NULL)) {
+#if PICO_STDIO_USB_SUPPORT_CHARS_AVAILABLE_CALLBACK
+        uint32_t chars_avail;
+#endif
         tud_task();
+#if PICO_STDIO_USB_SUPPORT_CHARS_AVAILABLE_CALLBACK
+        chars_avail = tud_cdc_available();
+#endif
         mutex_exit(&stdio_usb_mutex);
+#if PICO_STDIO_USB_SUPPORT_CHARS_AVAILABLE_CALLBACK
+        if (chars_avail) chars_available_callback(chars_available_param);
```

Comment:
The previous code did `if (chars_available_callback) {` before actually calling `chars_available_callback` - does the same thing need to happen here too?